spec/services/users/delete_account_service_spec.rb

```diff
@@ -130,10 +139,16 @@
         expect(audit_record.audit_type).to eq(
           AuditRecord.audit_types[:delete_account]
         )
+        expect(audit_record.metadata["name"]).to eq(user.name)
         expect(audit_record.metadata["email"]).to eq(user.email)
         expect(
           audit_record.metadata["account_deletion_notification_sent_at"]
         ).to eq(user.account_deletion_notification_sent_at.iso8601)
+
+        expect(audit_record.metadata["cohort_ids"]).to eq(
+          [student.cohort.id, student_2.cohort.id]
```

Comment:
```suggestion
          [student.cohort_id, student_2.cohort_id]
```
There's no need to load the cohort record just to get its id

Reply:
I forget it every time😂